jekyll/_cci1/index.md

```diff
@@ -5,39 +5,39 @@ title: "CircleCI 1.0 Documentation"
 permalink: /1.0/
 ---
 
-{% for category in site.data.categories %}
-{% if category.section == "cci1" %}
-
-{% if category.index %}
-	{% assign catFile = category.index %}
-{% else %}
-	{% assign catFile = category.slug %}
-{% endif %}
-
-<div class="category-section">
-	<img src="{{ site.baseurl }}/assets/img/icons/{{ category.icon }}" class="logo" alt="Category Icon" />
-	<h2>{{ category.name }}</h2>
-	<ul class="list-unstyled">
-	{% assign docs_found = 0 %}
-	{% assign sortedDocs = site.cci1 | sort: 'order' %}
-	{% for doc in sortedDocs %}
-		{% if doc.categories contains category.slug and doc.slug != catFile and doc.hide != true %}
-			{% assign docs_found = docs_found | plus: 1 %}
-			{% if docs_found < 4 %}
-				{% if doc.short-title %}
-					<li class="{% if page.path contains doc.url %}active{% endif %}"><a href="{{ site.baseurl }}{{ doc.url }}">{{ doc.short-title }}</a></li>
-				{% else %}
-					<li><a href="{{ site.baseurl }}{{ doc.url }}">{{ doc.title }}</a></li>
-				{% endif %}
-			{% endif %}
-		{% endif %}
-	{% endfor %}
-
-	{% if docs_found > 3 %}
-		{% assign more = docs_found | minus: 3 %}
-		<li><em><a href="{{ site.baseurl }}/1.0/{{ catFile }}/">{{ more }} more</a></em></li>
-	{% endif %}
-	</ul>
-</div>
-{% endif %}
-{% endfor %}
+CircleCI is a modern continuous integration and continuous delivery (CI/CD) platform. The hosted solution is available at <https://circleci.com/>. The [CircleCI Enterprise](https://circleci.com/enterprise/) solution is installable inside your private cloud or data center. Both are free to try for two weeks. If you are new to CircleCI, check out the [Getting Started with CircleCI]({{ site.baseurl }}/1.0/getting-started/).
```

Comment:
.com is 'free forever' for Linux with one container: https://circleci.com/pricing/